python/ray/job_configs.py

```diff
@@ -0,0 +1,45 @@
+import ray
+
+
+class JobConfigs:
+    """A class used to store the configurations of a job.
+
+    Attributes:
+        num_initial_python_workers (int): The initial Python workers to start
+            per node. If a negative value is specified, it fallbacks to
+            `num_cpus`.
+        num_initial_java_workers (int): The initial Java workers to start per
+            node. If a negative value is specified, it fallbacks to
+            `num_cpus`.
+        worker_env (dict): Environment variables to be set on worker
+            processes.
+        num_java_workers_per_process (int): The number of java workers per
+            worker process.
+        jvm_options (str[]): The jvm options for java workers of the job.
+    """
+
+    def __init__(
+            self,
+            num_initial_python_workers=-1,
+            num_initial_java_workers=-1,
+            worker_env=dict(),
```

Comment:
We should avoid using mutable values as default. https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
Thanks! Done.